src/components/WorkOrder/AssetTypeField.js

```diff
@@ -80,17 +80,23 @@ export default class AssetTypeField extends Component {
 
   componentDidMount() {
     this.setState({ loading: true });
-    let userPosition = {};
-    navigator.geolocation.getCurrentPosition(pos => {
-      userPosition["lat"] = pos.coords.latitude;
-      userPosition["lon"] = pos.coords.longitude;
 
-      getSignalsOptions(userPosition).then(data => {
-        this.setState({
-          signalOptions: data,
-          loading: false,
-          userPosition: userPosition,
-        });
+    let watchPosition = {};
+    navigator.geolocation.watchPosition(pos => {
```

Comment:
Let's assign this a value like `this.positionWatchEvent` so we can do 
`positionWatchEvent.clearWatch()` on `componentDidUnmount`

https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/clearWatch

Reply:
Glad that we paired on this - added before merge ✅